pipeline_dp/budget_accounting.py

```diff
@@ -8,139 +9,149 @@
 
 
 @dataclass
-class Budget:
-    """Manages the budget allocated for an operation.
+class MechanismSpec:
+    """Specifies the parameters for a mechanism.
 
-    The values for eps and delta are computed when the method compute_budgets
-    of the corresponding BudgetAccount is called.
+    NoiseKind defines the kind of noise distribution.
+    noise is the minimized noise standard deviation.
```

Comment:
noise -> _noise_standard_deviation

Please add in comments
(eps, delta) are parameters of (eps, delta)-differential privacy

Reply:
Done.